app/controllers/process_plates_controller.rb

```diff
@@ -36,9 +34,6 @@ def create
     # 20 barcodes takes about 2 mins as of 2020-06-22. This limit is to prevent the request timing out.
     raise "Please scan #{ProcessPlate::RECEIVE_PLATES_MAX} barcodes or fewer" if barcodes.count > ProcessPlate::RECEIVE_PLATES_MAX
 
```

Comment:
This limit was added due to wrangler performance issues, so can also be removed.